File: Code/cpp/quadratic_roots.cpp
#include<iostream>
#include<cmath>
using namespace std;
int main()
{
    float a, b, c, d;

    cout << "Enter Coefficient a: ";
    cin >> a;
    cout << "Enter Coefficient b: ";
    cin >> b;
    cout << "Enter Coefficient c: ";
    cin >> c;

    d = pow(b,2) - (4 * a * c);
    if (d > 0)
    {
        cout << (-b + sqrt(d)) / (2 * a) << "\n";
        cout << (-b - sqrt(d)) / (2 * a);
    }
    else if(d == 0)
        cout << -b / (2 * a);
    else
        cout << "Complex Roots";
    return 0;
}

File: Code/java/quadratic_roots.java
import java.util.Scanner;

class Roots
{
    public static void main(String[] args)
    {
        double a, b, c, d;
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Coefficient a: ");
        a = scanner.nextFloat();
        System.out.println("Enter Coefficient b: ");
        b = scanner.nextFloat();
        System.out.println("Enter Coefficient c: ");
        c = scanner.nextFloat();

        d = Math.pow(b, 2) - (4 * a * c);
        if (d > 0)
        {
            System.out.println((-b + Math.sqrt(d) )/ (2 * a));
            System.out.println((-b - Math.sqrt(d)) / (2 * a));
        }
        else if (d == 0)
            System.out.println(-b / (2 * a));
        else
            System.out.println("Complex Roots");

        scanner.close();
    }
}